components/member-card/show-wallet.js

```diff
@@ -0,0 +1,76 @@
+import classNames from 'components/member-card/card.module.scss';
+import { CURRENCIES, walletURL } from 'helper-functions/urls';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+const Coins = (props) => {
+  const { coin, balance } = props;
+  return (
```

Comment:
destructure the props directly on line no 6, and then we can omit the `return` statement and directly return it.

```javascript
const Coin = ({ coin, balance }) => (
 <div>...code</div>
);
```